exercises/anagram/src/test/java/AnagramTest.java

```diff
@@ -34,14 +34,6 @@ public void testDetectLongerAnagram() {
         assertThat(anagrams, hasItem("inlets"));
     }
 
-    @Ignore("Remove to run test")
-    @Test
-    public void testDetectMultipleAnagrams() {
```

Comment:
Note to self: This commit should be deleted because there's a difference between both tests. `testDetectMultipleAnagrams()` actually test for 2 matching anagrams, while `testDetectMultipleAnagramsForLongerWord()` test for 3 matching anagrams. As such, we should rename the method names instead for clarity.